File: FlareSDKJavaSample/app/src/main/java/com/example/myapplication/TestIncidentActivity.java
package com.example.myapplication;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageView;
import androidx.appcompat.app.AppCompatActivity;

import com.sos.busbysideengine.BBSideEngine;
import com.sos.busbysideengine.Constants;
import com.sos.busbysideengine.rxjavaretrofit.network.model.BBSideEngineListener;

import org.json.JSONObject;

public class TestIncidentActivity extends AppCompatActivity {

    ImageView ivCUIClose;
    @SuppressLint("ResourceAsColor")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test_incident);
        ivCUIClose = findViewById(R.id.ivCUIClose);
        ivCUIClose.setOnClickListener(v -> {
            finish();
        });
    }

    @Override
    protected void onStop() {
        super.onStop();
        finish();
    }

    @Override
    protected void onResume() {
        super.onResume();
    }

    @Override
    public boolean onNavigateUp() {
        finish();
        return true;
    }
}

File: SideEngineKotlinSample/app/src/main/java/com/app/flaresdkimplementation/TestIncidentActivity.kt
package com.app.flaresdkimplementation

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class TestIncidentActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test_incident)

        val ivCUIClose = findViewById<View>(R.id.ivCUIClose)
        ivCUIClose.setOnClickListener {
            finish()
        }
    }

    override fun onStop() {
        super.onStop()
        finish()
    }

    override fun onNavigateUp(): Boolean {
        finish()
        return true
    }
}